reset.sh

```diff
@@ -34,10 +34,16 @@ for volume in $(docker volume list --format '{{ .Name }}' | grep sentry); do
     || echo "Skipped volume: $volume"
 done
 
-# If we have a version (or other git ref) given, install it.
-ref="${1:-}"
-if [ -z "$ref" ]; then
-  exit
+# If we have a version given, switch to it.
+
+# Note that arbitrary git refs won't work, because the *_IMAGE variables in
+# .env will almost certainly point to :latest. Release tags of onpremise are
+# generally the only refs where these component versions are pinned.
+
+version="${1:-}"
```

Comment:
Let's ensure that `$version` is actually a known tag here (in case people try to pass branches or refs):
```shell
git rev-parse --verify -q refs/tags/$version
```

Reply:
Done in 2f5c171059500b6b10a71659ee68c17bb0876e46 and 61b5de2e7aeef55c0a7c4444938ca393a118f97f.